core/src/test/java/tachyon/client/BlockInStreamTest.java

```diff
@@ -55,6 +55,10 @@ public static final void beforeClass() throws IOException {
    */
   @Test
   public void readTest1() throws IOException {
+    if (sLocalTachyonCluster.isDummyUnderFS()) {
```

Comment:
It would be much better to replace this `if` clause with a JUnit assumption instead e.g.

   Assume.assumeFalse(sLocalTachyonCluster.isDummyUnderFS());

Just returning makes the test as passed which isn't really accurate because they are not being run, using a JUnit assumption causes JUnit to mark the test as skipped if the assumption fails which gives a more accurate picture of which tests were run